requirements.txt

```diff
@@ -1,4 +0,0 @@
-pandas
-numpy
-sklearn
-pytest
```

Comment:
Are there particular versions of these packages that you are using? It might be good to add those versions to the `requirements.txt`.